### DISCUSSION TITLE
Global demands list

### DISCUSSION THREAD

**Discussion by Antaguana** [voxeltycoon/issues#477] [General]
**Is your feature request related to a problem? Please describe.**
Trying to find industries means a lot of map searching. 

**Describe the solution you'd like**
A UI to list industries, filter by type, sort by name or demand.

**Describe alternatives you've considered**
currently, we need to go on a hunt to find an industry that is not being serviced well.

**Additional context**
See OTTD or Transport Fever industry list.

**Comment by katepolak** [comment]
Changed the title to better reflect the games terminology
